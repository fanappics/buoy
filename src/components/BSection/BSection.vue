<template>
  <header>
    <slot></slot>
  </header>
</template>

<script>
export default {
  name: 'b-section'
}
</script>

<style scoped>
header {
  background-color: rgb(248,248,248);
  display: flex;
  flex: 0 0 3rem;
  flex-direction: column;
  font-weight: bold;
  justify-content: center;
  padding: 0rem 2rem;
}
</style>
