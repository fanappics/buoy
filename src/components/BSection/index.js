import BSection from './BSection.vue'

/* istanbul ignore next */
BSection.install = function install (Vue) {
  Vue.component(BSection.name, BSection)
}

export default BSection
