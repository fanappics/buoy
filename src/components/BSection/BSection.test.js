import { mount } from 'vue-test-utils'
import BSection from './BSection'

describe('BSection', () => {
  it('renders correctly', () => {
    const slot = `<div />`
    const wrapper = mount(BSection, {
      slots: { default: slot }
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
