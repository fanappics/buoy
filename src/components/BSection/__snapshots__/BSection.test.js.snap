// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BSection renders correctly 1`] = `
<header>
    <div></div>
</header>
`;
